.rubocop.yml

```diff
@@ -94,3 +98,6 @@ RSpec/NotToNot:
 # Following this rule's suggestions breaks postgres support
 Sequel/ConcurrentIndex:
   Enabled: false
+
+Gemspec/RequireMFA:
```

Comment:
We should require MFA for publishing the gem, but I didn't want to do that as part of this PR.